cli/README.md

```diff
@@ -1,55 +1,74 @@
-# Xata CLI
+# Command Line Interface (CLI)
 
-## Installing
+Xata provides a CLI as a convenience for developers. It is intended to provide feature parity with Xata's [Web UI](https://docs.xata.io/getting-started) (User Interface) to enable developers to build applications at **rapid pace** with **as little friction as possible**.
 
-The recommended way of installing the CLI is by installing it as an npm package globally:
+## Installation
 
-```bash
-npm install @xata.io/cli -g
+Our CLI is distributed both as an **[npm](https://www.npmjs.com/package/@xata.io/cli) package** for developers who work with Node.js, and a **cross-platform binary** for those who don't. Below, you'll find commands that you can copy and paste into your platform's shell that will help you install the CLI on your specific platform.
+
+![Installing the Xata CLI](/screenshots/CliInstall.gif)
+
+### Installing with npm
+
+To install the Xata CLI globally, run the following command:
+
+```sh
+npm i --location global @xata.io/cli
```

Comment:
```suggestion
npm install -g @xata.io/cli
```

I think the `--location global` part is not very common. I had never seen it before.

Reply:
It's the current preferred way to install globally:

<img width="1011" alt="image" src="https://user-images.githubusercontent.com/9947422/177975111-3a148555-690d-472d-a4b2-f5039f26ad71.png">